File: php/samples/SimpleOrm.Sample/src/Migrations/V0003.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Sample\Migrations\Table\Transaction\V0003_AddNote;

final class V0003 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version
            ->apply(V0003_AddNote::class);
    }
}

File: go/orm/sample/migrations/V0003.go
package migrations

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample/migrations/Table/Transaction"
)

type V0003 struct{}

func (V0003) Compose(version *orm.VersionBuilder) {
	version.Apply(transaction.V0003_AddNote{})
}
